bin/win/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,33 @@
+﻿$ErrorActionPreference = 'Stop';
+$toolsPath = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$version = $env:chocolateyPackageVersion
+$pp = Get-PackageParameters
+
+if ($pp['path'] -ne $null){
```

Comment:
@drholmie thanks for this PR. There were a couple of failures in the tests:

Looks like we need a `then` statement here: https://github.com/koalaman/shellcheck/wiki/SC1050

Reply:
Hi there! Thanks for reviewing the PR :) So since this is a PowerShell script, shellcheck isn't actually meant for linting this. Although thanks for catching this. I completely missed linting the code. For PowerShell scripts [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) can be used, doing so (for default values) I got the following output:
```
RuleName                            Severity     ScriptName Line  Message
--------                            --------     ---------- ----  -------
PSAvoidTrailingWhitespace           Information  chocolatey 33    Line has trailing whitespace
                                                 install.ps
                                                 1
PSPossibleIncorrectComparisonWithNu Warning      chocolatey 7     $null should be on the left side of equality
ll                                               install.ps       comparisons.
                                                 1
PSPossibleIncorrectComparisonWithNu Warning      chocolatey 18    $null should be on the left side of equality
ll                                               install.ps       comparisons.
```
I'll make those changes however before adding to the PR, how should we go about adding PSScriptAnalyzer to the static checks of Linkerd? (if it is required to be added that is) or is it alright for now to just add `! -name win \` in the `static_check.yml` ?

Reply:
this is a good question. I found a [PSScriptAnalyzer GitHub Actions integration](https://github.com/devblackops/github-action-psscriptanalyzer).

@kleimkuhler @alpeb Is this something that we want to integrate into github actions? The option is to ignore the .ps1 script, per the latest commit.

Reply:
@cpretzer Definitely :+1: @drholmie can you take care of adding that as part of this PR? If you're not familiar with Github actions, I can take care of it as a followup PR.
Mind thought that'd be a separate effort from adding the chocolatey package into the Linkerd release assets, which I think should be tackled in yet another separate PR.

Reply:
Thank you all for reviewing this! As mentioned in the draft PR comment (https://github.com/linkerd/linkerd2/pull/3921#issuecomment-599634476). It would be better to handle this in a separate PR. Considering the scope of this PR is meant to add the required files for building the choco package in the `bin` folder.

Regarding adding this to the release process, @ihcsim has already done some work on this as seen [here](https://github.com/ihcsim/linkerd-chocolatey).